File: Exercises/Is_Number_Prime/Java/src/main/java/Prime.java
public class Prime {
    public static boolean isPrime(int num) {
        if (num <= 0 || num == 1) {
            return false;
        } else if (num == 2) {
            return true;
        } else if (num % 2 == 0) {
            return false;
        }
        for (int i = 3; i * i <= num; i += 2) {
            if (num % i == 0) {
                return false;
            }
        }
        return true;
    }
}

File: Exercises/Is_Number_Prime?/Python/prime.py
def is_prime(num):
    if num <= 0 or num == 1 or num == -1:
        return False
    if num == 2:
        return True
    if num % 2 == 0:
        return False

    i = 3

    for i in range(2, num):
        if num % i == 0:
            return False
        i += 2
    return True

print(is_prime(-5))
